addons/editor-devtools/blockly/BlockFlasher.js

```diff
@@ -9,20 +9,19 @@ export default class BlockFlasher {
   static flash(block) {
     if (myFlash.timerID > 0) {
       clearTimeout(myFlash.timerID);
-      myFlash.block.setColour(myFlash.colour);
+      myFlash.block.svgPath_.style.fill = "";
```

Comment:
Please elaborate the bug that this change (and others below it) fixes, related to theme3. Make an issue number for it and link it to this PR

Reply:
Problem is, devtools would set the blocks to their default color, and not it's custom. By setting its fill to "", it goes back to its custom color.